internal/operator-controller/applier/helm.go

```diff
@@ -232,3 +299,25 @@ func (p *postrenderer) Run(renderedManifests *bytes.Buffer) (*bytes.Buffer, erro
 	}
 	return &buf, nil
 }
+
+func ruleDescription(ns string, rule rbacv1.PolicyRule) string {
+	var sb strings.Builder
+	sb.WriteString(fmt.Sprintf("Namespace:%q", ns))
+
+	if len(rule.APIGroups) > 0 {
+		sb.WriteString(fmt.Sprintf(" APIGroups:[%s]", strings.Join(rule.APIGroups, ",")))
```

Comment:
would we want to sort each of rule.APIGroups, rule.Resources, etc. before joining? e.g. something like:
```suggestion
		sb.WriteString(fmt.Sprintf(" APIGroups:[%s]", strings.Join(slices.Sorted(slices.Values(rule.APIGroups)), ",")))
```

Reply:
done in 13019720